aion/web/elm/src/View.elm

```diff
@@ -5,13 +5,70 @@ import Html.Attributes exposing (..)
 import Html.Events exposing (onInput)
 
 import Msgs exposing (Msg(..))
-import Models exposing (Model)
+import Models exposing (Model, Room)
 import String exposing (concat)
+import Routing exposing (roomsPath)
+
 
 view: Model -> Html Msg
 view model =
   div []
+    [ page model ]
+
+
+page: Model -> Html Msg
+page model =
+  case model.route of
+    Models.LoginRoute ->
+      loginView model
+
+    Models.RoomsRoute ->
+      roomsView model
+
+    Models.NotFoundRoute ->
+      notFoundView
+
+
+-- LOGIN VIEW
+loginView: Model -> Html Msg
+loginView model =
+  div []
     [ p [] [text "Welcome to Aion!"]
-    , input [ type_ "text", placeholder "Username", onInput Username] []
+    , input [ type_ "text", placeholder "Username", onInput UpdateUsername] []
     , p [] [text (concat ["Hello ", model.username, ""])]
+    , navigationButton
+    ]
+
+
+navigationButton: Html Msg
+navigationButton =
+  let
+    path = roomsPath
+  in
+    a
+      [ href path ]
+      [ i [] []
+      , text "Rooms"]
+
+
+-- ROOMS VIEW
+roomsView: Model -> Html Msg
+roomsView model =
+  div []
+    [ div [ class "roomsNames" ] [ text "Rooms:" ]
```

Comment:
I would stick to the 'roomNames' variable name - one noun in plural says it all.